crates/subspace-service/src/chain_spec.rs

```diff
@@ -14,16 +14,15 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+//! Subspace chain configurations.
```

Comment:
Why is chain-spec moved into service crate?

Reply:
I simply referred to the design of polkadot-service when extracting subspace-service.